### DISCUSSION TITLE
Multiple HTML for tailwinds

### DISCUSSION THREAD

**Discussion by AppleCraft2005** [tailwindlabs/tailwindcss#13742] [General]
So i have plan to build a simple web using tailwind. The website are consist of multiple HTML file and some JS. My question is, is it okay for all the HTML file to connect to a single style.css file ??![IMG_20240526_080002.jpg](https://github.com/tailwindlabs/tailwindcss/assets/141586230/e8a1956e-8686-4560-83b1-6a3ec7e18512)

Like in the photo i got three HTML file and all of it connect  in to style.css

**Comment by wongjn** [comment]
Yeo! I believe that's the way Tailwind should be used.

**Reply by AppleCraft2005** [reply]
Alrigt then thanks for the answer 👍👍

**Comment by WayneF11** [comment]
Yes, it’s perfectly fine to link multiple HTML files to a single style.css file. This is a common practice and helps maintain consistency in design across different pages. Just ensure that your CSS is well-organized and uses classes efficiently to style elements appropriately for each page. [Publix Passport](https://www.publix-passport.com)

**Reply by AppleCraft2005** [reply]
Thanks for the reply mate appreciate that 🔥🔥
